engine/include/cubos/engine/render/camera/camera.hpp

```diff
@@ -0,0 +1,27 @@
+/// @file
+/// @brief Component @ref cubos::engine::Camera
+/// @ingroup render-camera-plugin
+
+#pragma once
+
+#include <cubos/core/reflection/reflect.hpp>
+#include <cubos/core/reflection/external/glm.hpp>

```

Comment:
You only need this header in the `.cpp`, for the `CUBOS_REFLECT_IMPL`.
We try to reduce the headers we have in the headers as much as possible in order to keep compile times as short as possible.